common/djangoapps/student/tests/test_refunds.py

```diff
@@ -77,26 +77,6 @@ def test_refundable_expired_verification(self, cutoff_date):
         self.verified_mode.save()
         self.assertTrue(self.enrollment.refundable())
 
-    def test_refundable_of_purchased_course(self):
```

Comment:
What is the logic behind removing this test . Are you going to move this to some where else. We need a test here.